model/articles.models.js

```diff
@@ -11,4 +11,16 @@ function fetchArticleById(article_id) {
   });
 }
 
-module.exports = { fetchArticleById };
+function fetchArticle(sort_by = "created_at", order = "desc") {
+  let queryInsert = `SELECT articles.article_id, articles.author, title, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.body) AS comment_count 
```

Comment:
You've mentioned comment_count here, but you haven't expected it in your test. How do you know it's being returned?